aggregator/apps/api/v1/stitcher.py

```diff
@@ -150,6 +150,7 @@ def make_result_known_response(self):
                 ballot["election_id"] = wcivf_ballot["election_id"]
                 ballot["election_name"] = wcivf_ballot["election_name"]
                 ballot["post_name"] = wcivf_ballot["post"]["post_name"]
+                ballot["candidates_final"] = wcivf_ballot["ballot_locked"]
```

Comment:
"Confirmed"? "Verified"? 

I think "final" doesn't work so well, but can't quite put my finger on why. I agree "locked" isn't great in this context either.

This is a future feature request, but you can imagine one day we have a "membership status" concept that has some values like "self-declared", "official party assertion", "SOPN verified" or whatever (more naming fun!). What we're really doing here is explaining to someone the membership status, and some people will only want to show candidates to users if they've been verified from the SOPN.

This is also something we could open up to other API users / Slack members for feedback – our mental models will change the way we think about this.

Reply:
> "Confirmed"? "Verified"?

Yeah I like verified - have gone with that :+1:

> you can imagine one day we have a "membership status" concept that has some values like "self-declared", "official party assertion"

I see where you're going, but I don't think it changes anything here though because that would be a property of each candidacy, as opposed to an assertion about the ballot/candidate list. At that point it would move to being a field on the person/candidacy. At the ballot-level I think the only status that makes sense is: "this list is the same list as the SOPN" or "it isn't/we haven't checked it yet"